### ISSUE TITLE
incompatibility with bs-platform >= 4.0.6

### ISSUE THREAD

**Issue by codekiln** [wyze/bs-jest-dom#2]
Steps to reproduce: 

* clone `bs-jest-dom` 
* ~remove `yarn.lock` & run `yarn install`~ run `yarn add bs-platform@4.0.6`
* run `yarn test` to trigger compilation

Error: 

```
  We've found a bug for you!
  /Users/pnore/dev/bs-jest-dom/src/JestDom.re
  
  The implementation src/JestDom.mlast
       does not match the interface src/jestDom.cmi:
       The value `not__' is required but not provided
```

This is from the change mentioned in the [glennsl/bs-jest#41 issue included in 0.4.7](https://github.com/glennsl/bs-jest/issues/41), which I think was introduced with [`bs-platform@4.0.6`](https://github.com/BuckleScript/bucklescript/blob/master/Changes.md#406).

**Comment by wyze** [comment]
Yes, I ran into this as well when I tried to upgrade, which is still in my local repo and filed an issue here: https://github.com/BuckleScript/bucklescript/issues/3094. Looks like there is a workaround now so I should be able to get that fixed up.

**Comment by wyze** [comment]
This should be fixed in the latest version I just released ([v2.0.0](https://github.com/wyze/bs-jest-dom/releases/tag/v2.0.0)). I actually rewrote the library to better interface with `bs-jest`, so now we don't have our own `expect`/`not_` operators.
